pkg/system/system.go

```diff
@@ -401,6 +402,70 @@ func RunYaml(cmd *cobra.Command, args []string) {
 	util.Panic(p.PrintObj(sys, os.Stdout))
 }
 
+// CheckNooBaaImages runs a CLI command
+func CheckNooBaaImages(cmd *cobra.Command, sys *nbv1.NooBaa, args []string) string {
+	log := util.Logger()
+
+	noobaaImage := *sys.Spec.Image
+	noobaaActualImage := sys.Status.ActualImage
+	noobaaCoreStatefulset := util.KubeObject(bundle.File_deploy_internal_statefulset_core_yaml).(*appsv1.StatefulSet)
+	noobaaCoreStatefulset.Namespace = options.Namespace
+	util.KubeCheckQuiet(noobaaCoreStatefulset)
+	noobaaCoreStatefulsetImage := noobaaCoreStatefulset.Spec.Template.Spec.Containers[0].Image
+	if noobaaActualImage != noobaaCoreStatefulsetImage {
```

Comment:
I don't think spec.Image 
```suggestion
	desiredImage := ""
	runningImage := ""
	if sys.Status.ActualImage != "" {
		desiredImage = sys.Status.ActualImage
	} else if sys.Spec.Image != nil {
		desiredImage = *sys.Spec.DBImage
	}
	sts := util.KubeObject(bundle.File_deploy_internal_statefulset_core_yaml).(*appsv1.StatefulSet)
	sts.Namespace = options.Namespace
	if util.KubeCheckQuiet(sts) {
		runningImage = sts.Spec.Template.Spec.Containers[0].Image
	}
	if desiredImage != "" && runningImage != "" && desiredImage != runningImage {
```